consensus/service/src/byzantine_ledger/worker.rs

```diff
@@ -556,8 +556,6 @@ impl<
                 self.tx_manager.remove_expired(index)
             };
 
-            counters::TX_CACHE_NUM_ENTRIES.set(self.tx_manager.num_entries() as i64);
```

Comment:
this is duplicated, because the tx_manager `.remove_expired` already sets this value